main.go

```diff
@@ -661,6 +644,35 @@ func transformCompile(args []string) ([]string, error) {
 		}
 		files = append(files, file)
 	}
+	tf := newTransformer()
+	if err := tf.typecheck(files); err != nil {
+		return nil, err
+	}
+
+	if err := loadKnownReflectAPIs(); err != nil {
+		panic(err)
```

Comment:
return err?

Reply:
This shouldn't happen so I think panic is appropriate